src/main/resources/reference.conf

```diff
@@ -1327,12 +1330,6 @@ resourceTypes = {
       discoverer = {
         roleActions = ["discover_data", "read_policy::steward", "read_policy::discoverer", "read_auth_domain", "get_snapshot_builder_settings"]
       }
-      snapshot_builder_manager = {
-        roleActions = ["update_snapshot_builder_settings"]
```

Comment:
Can we also remove the "update_snapshot_builder_settings" action?

Reply:
Ah, I see your comments - Ok, my PR is blocking removing this.

Reply:
Oh, I was going to say I think we can. Its `datasets` which still use `update_snapshot_builder_settings`